Tests/ApolloTests/ParseQueryResponseTests.swift

```diff
@@ -46,15 +46,9 @@ class ParseQueryResponseTests: XCTestCase {
       ]
     ])
 
-    XCTAssertThrowsError(try response.parseResult().await()) { error in
-      if let error = error as? GraphQLResultError, case JSONDecodingError.couldNotConvert(let value, let expectedType) = error.underlying {
-        XCTAssertEqual(error.path, ["hero", "name"])
-        XCTAssertEqual(value as? Int, 10)
-        XCTAssertTrue(expectedType == String.self)
-      } else {
-        XCTFail("Unexpected error: \(error)")
-      }
-    }
+    let (result, _) = try response.parseResult().await()
+
+    XCTAssertEqual(result.data?.hero?.name, "10")
```

Comment:
Why shouldn't this throw an error when it's not a string? `name` isn't using a custom scalar type

Reply:
You are absolutely right. I will check this test case again and come with a better solution.

Reply:
Punting on this until later - we'll figure it out at some point.